docs/src/main/asciidoc/liquibase.adoc

```diff
@@ -0,0 +1,219 @@
+            <column name="NAME" type="VARCHAR(255)"/>
+        </createTable>
+    </changeSet>
+</databaseChangeLog>
+--
+
+Now you can start your application and Quarkus will run the Liquibase's update method according to your config:
+
+[source,java]
+--
+import org.quarkus.liquibase.Liquibase <1>
+
+@ApplicationScoped
+public class MigrationService {
+    // You can Inject the object if you want to use it manually
+    @Inject
+    Liquibase liquibase; <2>
+
+    public void checkMigration() {
+        // Get the list of liquibase change set statuses
+        List<ChangeSetStatus> status = liquibase.getChangeSetStatuses();
+    }
+}
+--
+
+<1> The Quarkus extension wraps the Liquibase object in its own implementation
+<2> Inject the Quarkus liquibase object if you want to use the liquibase methods directly
+
+== Multiple datasources
+
+Liquibase can be configured for multiple datasources.
+The Liquibase properties are prefixed exactly the same way as the named datasources, for example:
+
+[source,properties]
+--
+quarkus.datasource.driver=org.h2.Driver
+quarkus.datasource.url=jdbc:h2:tcp://localhost/mem:default
+quarkus.datasource.username=username-default
+quarkus.datasource.min-size=3
+quarkus.datasource.max-size=13
+
+quarkus.datasource.users.driver=org.h2.Driver
+quarkus.datasource.users.url=jdbc:h2:tcp://localhost/mem:users
+quarkus.datasource.users.username=username1
+quarkus.datasource.users.min-size=1
+quarkus.datasource.users.max-size=11
+
+quarkus.datasource.inventory.driver=org.h2.Driver
+quarkus.datasource.inventory.url=jdbc:h2:tcp://localhost/mem:inventory
+quarkus.datasource.inventory.username=username2
+quarkus.datasource.inventory.min-size=2
+quarkus.datasource.inventory.max-size=12
+
+# Liquibase configuration for the default datasource
+quarkus.liquibase.schemas=DEFAULT_TEST_SCHEMA
+quarkus.liquibase.change-log=db/changeLog.xml
+quarkus.liquibase.migrate-at-start=true
+
+# Liquibase configuration for the "users" datasource
+quarkus.liquibase.users.schemas=USERS_TEST_SCHEMA
+quarkus.liquibase.users.change-log=db/users.xml
+quarkus.liquibase.users.migrate-at-start=true
+
+# Liquibase configuration for the "inventory" datasource
+quarkus.liquibase.inventory.schemas=INVENTORY_TEST_SCHEMA
+quarkus.liquibase.inventory.change-log=db/inventory.xml
+quarkus.liquibase.inventory.migrate-at-start=true
+--
+
+Notice there's an extra bit in the key.
+The syntax is as follows: `quarkus.liquibase.[optional name.][datasource property]`.
+
+NOTE: Without configuration, Liquibase is set up for every datasource using the default settings.
+
+== Using the Liquibase object
+
+In case you are interested in using the `Liquibase` object directly, you can inject it as follows:
+
+NOTE: The Liquibase object is the instance of the org.quarkus.liquibase.Liquibase class. This kind of wrapper object to handle the
+JDBC connection for the liquibase.Liquibase object. The instance open and close the database connection for each method separately.
```

Comment:
This wording is a bit confusing.

Reply:
What would you suggest?
It is deep technical point and for guides is not so relevant, I can remove this note if you are ok with that.

Reply:
Better move this to the bullet point description (<1>), maybe something along the lines of:

```
NOTE: As a technical detail, the injected `Liquibase` object is an `org.quarkus.liquibase.Liquibase` instance, which acts as a wrapper for the underlying JDBC connection and is used to open and close the database connection for each method separately

Reply:
Like this?
![image](https://user-images.githubusercontent.com/828045/72737943-d4d8b700-3ba0-11ea-90eb-c498fdbefb76.png)

Reply:
yeah, that looks cool